decidim-core/app/views/layouts/decidim/_application.html.erb

```diff
@@ -5,6 +5,7 @@
     <title><%= decidim_page_title %></title>
     <%= render partial: "layouts/decidim/head" %>
     <%= render partial: "layouts/decidim/js_configuration" %>
+    <link rel="manifest" href="/manifest.webmanifest">
```

Comment:
@Crashillo move this to `layouts/decidim/head` partial

Reply:
Moved